frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -398,9 +444,13 @@ function VirtualKeyPopup:init()
                     h_key_padding[i].width = h_key_padding[2].width
                 end
 
+                local key = v.key or v
```

Comment:
That could be dangerous (even if the below isn't currently expected), e.g.:

```
> v = true
> print(v.lala)
stdin:1: attempt to index global 'v' (a boolean value)
stack traceback:
        stdin:1: in main chunk
        [C]: in ?
```

It's probably more future proof to write something like:
```lua
type(v) == "table" and v.key or v
```